commands/RuneScape - Fun/raid.js

```diff
@@ -0,0 +1,189 @@
+const { Command } = require('klasa');
+const RaidsEmojis = require('../../resources/monsters/raids').drops;
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			cooldown: 1,
+			description: 'Roll on the raids drop table based on the number of points specified',
+			usage: '<points:int{1}>',
+			usageDelim: ' '
+		});
+	}
+
+	async run(msg, [points]) {
+		return msg.send(RaidsUtil.raid(points));
+	}
+
+};
+
+
+class RaidsUtil {
+
+	static raid(points) {
+		const chances = RaidsUtil.calculateChance(points);
+		const loot = RaidsUtil.rollLoot(chances);
+		return RaidsUtil.formatReply(points, chances, loot);
+	}
+
+	static calculateChance(points) {
+		const chances = [];
+		let cumulativeChance = points * LootChance.perPoint;
+		for (let i = 0; i < LootChance.maxDrops; i++) {
+			if (cumulativeChance > LootChance.cap) {
+				chances.push(LootChance.cap);
+				cumulativeChance -= LootChance.cap;
+			} else {
+				chances.push(cumulativeChance);
+				break;
+			}
+		}
+		return chances;
+	}
+
+	static rollLoot(chances) {
+		return chances.map(chance => {
+			if (Math.random() < chance) {
+				const drop = RaidsUtil.pickFromTable();
+				drop.olmlet = RaidsUtil.rollOlmlet();
+				return drop;
+			}
+			return null;
+		});
+	}
+
+	static pickFromTable() {
+		const rnd = Math.floor(Math.random() * LootChance.totalWeight);
+		return UniqueDropWeights.find(i => i.shortName === LootTable[rnd]);
+	}
+
+	static formatReply(points, chances, loot) {
+		const plural = chances.length > 1 ? 's' : '';
+		const reply = [
+			`You finished a raid with ${points} points.`,
+			`You get ${chances.length} roll${plural} on the loot table.`
+		];
+
+		chances.forEach((chance, i) => {
+			const drop = loot[i] ? `${loot[i].name} ${RaidsEmojis[loot[i].shortName]}` : 'You no lucky';
+			const prefix = chances.length > 1 ? `Roll ${i + 1}:` : '';
+			const dropChance = Math.round(chance * 100);
+			reply.push(`${prefix} ${dropChance}% chance for a drop - ${drop}`);
```

Comment:
If there is only 1 roll, theres an extra space inserted at the beginning of this line
![untitled](https://user-images.githubusercontent.com/30398469/41808102-0b4bd060-771c-11e8-9887-ac619285b429.png)